components/General/EducadoModal.tsx

```diff
@@ -20,44 +26,34 @@ import PropTypes from "prop-types";
  * - title: String for modal title
  * @returns
  */
-export default function EducadoModal(props) {
+const EducadoModal = (props: EducadoModalProps) => {
```

Comment:
Use [object destructuring](https://javascript.info/destructuring-assignment#object-destructuring) for prop parameters.